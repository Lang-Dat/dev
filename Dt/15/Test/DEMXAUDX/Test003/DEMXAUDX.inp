167 178
XCXCCXXCCXXCXCXCCXCXCCXCCXCXXCXCCCCCCXXCXCCCXXXXXCXCXCXCXXCCXXCXCXCXCCXCCXXXCXCCCCXCCXXXXXXCCCCXXXXCXCCXXXXXXXCCCXCCCCXCCXCXXXCXCXCXCCXXCCXXXXXCCXXXXCCCCXXCCCCXCCCXXXX
15 150
11 160
4 146
3 150
81 85
155 162
149 150
8 156
155 160
14 156
8 89
97 138
11 164
27 161
7 129
11 144
3 133
153 166
16 127
16 146
162 167
5 124
14 135
39 80
80 94
10 92
90 156
13 158
3 167
7 145
6 126
13 155
11 132
42 71
6 124
111 142
75 144
70 142
16 126
15 150
13 137
101 121
9 17
106 106
13 159
87 167
2 167
20 145
9 152
160 162
37 125
60 124
16 163
5 120
81 113
2 122
103 109
16 143
2 138
87 123
148 158
11 146
2 132
156 156
153 162
13 150
5 140
118 158
10 165
143 153
13 115
32 36
125 146
3 167
20 105
13 74
13 159
2 55
114 135
1 127
119 131
124 141
4 129
118 132
7 151
90 115
87 129
54 120
16 152
123 139
112 118
120 128
94 103
3 127
108 114
99 111
155 163
160 162
16 126
10 166
167 167
6 36
8 119
10 130
80 125
132 160
13 130
49 60
8 149
146 161
10 163
6 25
2 161
4 137
90 107
159 167
1 128
62 114
100 160
2 157
66 142
13 159
2 147
16 150
22 152
12 122
72 93
10 128
146 158
7 128
80 106
1 117
14 124
104 146
127 132
10 145
20 93
139 163
13 120
14 160
4 124
1 166
91 160
45 122
9 52
10 149
86 146
7 139
16 162
12 122
165 165
15 137
66 112
5 121
6 119
3 123
15 118
7 163
10 119
135 164
7 149
136 167
13 156
14 152
73 130
68 141
11 120
56 101
1 136
89 114
25 94
9 143
15 148
2 150
120 127
5 147
8 141
160 161
