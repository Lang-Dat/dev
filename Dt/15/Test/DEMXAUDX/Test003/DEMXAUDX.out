405
451
421
439
9
18
2
442
13
426
233
119
462
403
361
398
387
33
330
387
13
354
361
117
42
242
197
436
491
409
356
427
363
89
351
87
207
211
328
405
368
65
18
1
441
246
493
371
429
4
262
189
443
343
102
357
22
379
400
113
27
404
385
1
22
412
398
116
466
27
306
15
59
491
246
181
441
149
56
371
29
45
370
36
430
74
128
194
408
41
16
18
21
367
14
39
22
4
328
469
1
85
332
358
138
86
349
31
423
44
462
50
475
392
44
20
374
156
185
461
222
441
430
402
389
332
59
352
34
358
75
344
330
129
12
403
213
75
322
441
357
491
209
238
121
419
185
389
439
332
1
361
137
347
337
358
310
468
328
89
425
94
429
415
170
213
329
127
397
76
202
400
398
441
15
422
393
2
